docs/en/02_Developer_Guides/18_Cookies_And_Sessions/02_Sessions.md

```diff
@@ -110,6 +110,19 @@ SilverStripe\Control\Session:
 
 This uses the session_name `SECSESSID` for `https` connections instead of the default `PHPSESSID`. Doing so adds an extra layer of security to your session cookie since you no longer share `http` and `https` sessions.
 
+## Relaxing checks around user agent strings
+
+Out of the box, SilverStripe will invalidate a user's session if the `User-Agent` header changes. This provides some supplemental protection against session high-jacking attacks.
```

Comment:
I didn't want people to get the impression, this was the primary mechanism against session high-jacking. I was trying to convey that disabling this is probably not a great idea, but not a gaping security hole either.